src/clj/rems/form_validation.clj

```diff
@@ -2,7 +2,12 @@
   "Pure functions for form validation logic"
   (:require [clojure.string :as str]
             [rems.common.form :as form]
-            [rems.common.util :refer [+email-regex+ +phone-number-regex+]]))
+            [rems.common.util :refer [+email-regex+
+                                      +phone-number-regex+
+                                      +valid-ip-address-regex+
+                                      +valid-ip-address-regex-version-six+
+                                      +reserved-ip-address-range-regex+
+                                      +reserved-ip-address-range-regex-version-six+]]))
```

Comment:
Minor point: I think we put the `+email-regex+` in `rems.common.util` so that it could be used in various places, e.g. r`ems.db.user-settings`. However the ip regexes could just live in this namespace since they'll probably not be used for anything else than form-validation.